tests/aws/services/cloudformation/resources/test_stepfunctions.py

```diff
@@ -216,6 +219,7 @@ def _sfn_finished_running():
     assert "hello_with_path from stepfunctions" in execution_result["output"]
 
 
+@pytest.mark.skip("Terminates with FAILED on cloud; convert to SFN v2 snapshot lambda test.")
 @markers.aws.unknown
```

Comment:
if it doesn't work against AWS please change `@markers.aws.unknown` to `@markers.aws.needs_fixing` :pray: